### PR TITLE
Fetch exchange rates for KSM and USDT

### PR DESCRIPTION
**Pull request by ashi20192020**
#373

### CODE REVIEW

File: src/mappings/mappingExchangeRateHandler.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+      value: usdtExchangeRate.toString()
+    };
+
+   const exchangeData: ExchangeData[] = [];
+   exchangeData.push(ksmExchangeData);
+   exchangeData.push(usdtExchangeData);
+
+   record.rates = exchangeData;
+
+    await record.save().catch(e => { console.log(e) });
```
Comment (0x8f701) [inline src/mappings/mappingExchangeRateHandler.ts]: error should be handled by caller instead, we can remove this catch

File: src/types/interfaces.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+// Auto-generated , DO NOT EDIT
+
+export interface ExchangeData {
+
+    key: string;
+
+    value: string;
+
+}
```
Comment (0x8f701) [inline src/types/interfaces.ts:11]: let's remove empty lines

File: src/types/models/ExchangeRate.ts [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    ExchangeData,
+} from '../interfaces'
+
+
+export class ExchangeRate implements Entity {
+
+    constructor(id: string) {
+        this.id = id;
+    }
+
```
Comment (0x8f701) [inline src/types/models/ExchangeRate.ts:15]: maybe you can add prettier to format code, you can refer to https://github.com/parallel-finance/stake-client

File: src/types/interfaces.ts (hunk 2 above)
Comment (ashi20192020) [reply src/types/interfaces.ts:11]: This is Auto-generated

File: package.json [hunk 4]
```diff
@@ -4,14 +4,14 @@
   "description": "",
   "main": "dist/index.js",
   "scripts": {
+    "install-prettier": "npm install --save-dev eslint-config-prettier",
```
Comment (0x8f701) [inline package.json]: we don't need this, it should be written in `dev-dependencies` if we need `eslint-config-prettie`

**Review by 0x8f701 — APPROVED** [review]

**Pull request merged**